src/olympia/devhub/templates/devhub/includes/policy_form.html

```diff
@@ -1,11 +1,9 @@
 {% from "devhub/includes/macros.html" import tip, some_html_tip %}
 <tr>
   <th>{{ tip(_('End-User License Agreement'),
-             _('Users will be required to accept the following End-User License Agreement (EULA) '
-               'prior to installing your add-on. The presence of a EULA significantly affects '
-               'the number of downloads an add-on receives. Please note that a EULA is not '
+             _('Please note that a EULA is not '
                'the same as a code license, such as the GPL or MPL.'
-               'It is only relevant for listed add-ons.')) }}</th>
+               ' It is only relevant for listed add-ons.')) }}</th>
```

Comment:
minor point, but we always have the space on the end of the previous string (so after the `MPL.`)

Reply:
@eviljeff Alright, got it! Thanks